cppcache/shared/CMakeLists.txt

```diff
@@ -41,11 +41,20 @@ target_include_directories(apache-geode
   PUBLIC
     $<TARGET_PROPERTY:_apache-geode,INTERFACE_INCLUDE_DIRECTORIES>)
 
-set_target_properties(apache-geode PROPERTIES
-  PUBLIC_HEADER "${PUBLIC_HEADERS}"
-  OUTPUT_NAME ${PRODUCT_LIB_NAME}
-  CXX_VISIBILITY_PRESET hidden
-  VISIBILITY_INLINES_HIDDEN ON)
+if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
Rather than duplicate the entire se_target_properties just change the single property for clang after setting the preferred properties.